### ISSUE TITLE
The mean distribution graph does not always show when multiple runs are specified

### ISSUE THREAD

**Issue by GoogleCodeExporter** [jeevandongre/namebench#86]
```
The google chart api is limited to 300000 pixels, thus if 720*"the calculated 
distribution's charts height" exceeds this, it should automatically be set to 
416 (or the width should be reduced to approx. 600 and the width to 500).
```

Original issue reported on code.google.com by `websiteb...@gmail.com` on 2 Jan 2010 at 9:34

**Comment by GoogleCodeExporter** [comment]
```
Thanks for the heads up. It's worth noting that anything more than the 2nd run 
is 
somewhat unnecessary, though namebench does nothing to prevent it.

```

Original comment by `thomas.r...@gmail.com` on 3 Jan 2010 at 11:14
- Changed state: **Accepted**
- Added labels: **Priority-Low**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
Issue 90 has been merged into this issue.
```

Original comment by `tstromb...@google.com` on 5 Jan 2010 at 3:17

**Comment by GoogleCodeExporter** [comment]
Original comment by `tstromb...@google.com` on 5 Jan 2010 at 3:18
- Changed title: **The mean distribution graph does not always show when multiple runs are specified**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 1.3b1.

```

Original comment by `helixblue` on 20 May 2010 at 3:27
- Changed state: **Fixed**
